buildlib/azure-pipelines-pr.yml

```diff
@@ -47,6 +47,26 @@ stages:
                  echo "##vso[task.complete result=Failed;]"
               fi
             condition: eq(variables['Build.Reason'], 'PullRequest')
+  - stage: Format
+    jobs:
+      # Check that the code is formatted according to the code style guidelines
+      - job: format
+        displayName: format code
+        container: fedora
+        steps:
+          - checkout: self
+            clean: true
+
+          - bash: |
+              git-format-diff remotes/origin/$(SYSTEM_PULLREQUEST_TARGETBRANCH)
```

Comment:
clang-format-diff ?

Reply:
Ignore - didn't realize that they have git utility.

Reply:
Actually its name is `git-clang-format`. Fixed, thanks.